airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/manifest.yaml

```diff
@@ -2192,7 +2192,7 @@ definitions:
         $ref: "#/definitions/report_creation_error_handler"
     polling_requester:
       type: HttpRequester
-      path: /reporting/reports/{{stream_slice['create_job_response'].json()['reportId']}}
+      path: /reporting/reports/{{ creation_response['id'] }}
```

Comment:
Tests aren't passing. It seems to be because it should be `creation_response['reportId']` here. There might be other underlying issues to fix too though